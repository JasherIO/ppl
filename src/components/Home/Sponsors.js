import React from 'react'
import { graphql, StaticQuery } from 'gatsby'
import { OutboundLink } from 'gatsby-plugin-google-analytics'

// const sponsors = [
//   {
//     'id': 1,
//     'title': 'All Yours Buffet',
//     'description': 'All the colored eggs you could ever want!',
//     'img': '/img/AllYours.png',
//     'href': 'https://google.com'
//   },
//   {
//     'id': 2,
//     'title': 'What a Save Esports',
//     'description': 'The cost cutting esports organization saving the world!',
//     'img': '/img/WhatASave.png',
//     'href': 'https://google.com'
//   },
//   {
//     'id': 3,
//     'title': 'Egging Grocers',
//     'description': 'Egg Grocers! Just do it! #JorbyApproved',
//     'img': '/img/EggingGrocers.png',
//     'href': 'https://google.com'
//   }
// ]

const Level = ({ sponsors }) => (
  <div className="level is-mobile">
    <div className="level-left">
      <div className="level-item">
        <p className="title is-4">Sponsored By</p>
      </div>
    </div>
    <div className="level-right">
      {sponsors.map((sponsor) => (
          <div key={sponsor.name} className="level-item">
            <figure className="image is-96x96 is-hidden-touch">
              <OutboundLink href={sponsor.link}>
                <img src={sponsor.img} alt={sponsor.name} />
              </OutboundLink>
            </figure>
            <figure className="image is-48x48 is-hidden-desktop">
              <OutboundLink href={sponsor.link}>
                <img src={sponsor.img} alt={sponsor.name} />
              </OutboundLink>
            </figure>
          </div>
        )
      )}
    </div>
  </div>
)

const Section = ({ data }) => {
  const { sponsors } = data.markdownRemark.frontmatter

  return (
    <section className="section is-small background is-hexellence">
      <div className="container">
        <Level sponsors={sponsors} />
      </div>
    </section>
  )
}

const query = graphql`
  query {
    markdownRemark(frontmatter: { templateKey: { eq: "sponsor-post" } }) {
      frontmatter {
        sponsors {
          name
          img
          link
        }
      }
    }
  }
`

export default props => (
  <StaticQuery query={query} render={data => <Section data={data} {...props} />} />
)