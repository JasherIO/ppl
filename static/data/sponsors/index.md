---
templateKey: sponsor-post
sponsors:
  - img: /img/AllYours.png
    link: 'https://google.com'
    name: All Yours Buffet
  - img: /img/WhatASave.png
    link: 'https://google.com'
    name: What a Save Esports
  - img: /img/EggingGrocers.png
    link: 'https://google.com'
    name: Egging Grocers
---

