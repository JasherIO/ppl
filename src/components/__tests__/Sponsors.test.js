import React from "react"
import renderer from "react-test-renderer"

import { PureSection, Level } from "../Sponsors"
import data from './__data__/home.json'

describe("Level", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<Level sponsors={data.markdownRemark.sponsors} />).toJSON()
    expect(tree).toMatchSnapshot()
  }),
  it("renders nothing", () => {
    const tree = renderer.create(<Level sponsors={[]} />).toJSON()
    expect(tree).toMatchSnapshot()
  }))

describe("Sponsor Section", () =>
  // Mock level

  it("renders correctly", () => {
    const tree = renderer
      .create(<PureSection data={data} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))