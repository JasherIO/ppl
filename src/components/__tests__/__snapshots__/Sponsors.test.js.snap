// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level renders correctly 1`] = `
<div
  className="level is-mobile"
>
  <div
    className="level-item"
  >
    <p
      className="title is-4"
    >
      Sponsored By
    </p>
  </div>
  <div
    className="level-item"
  />
</div>
`;

exports[`Sponsor Section renders correctly 1`] = `
<section
  className="section is-small"
>
  <div
    className="container"
  >
    <div
      className="level is-mobile"
    >
      <div
        className="level-item"
      >
        <p
          className="title is-4"
        >
          Sponsored By
        </p>
      </div>
      <div
        className="level-item"
      >
        <div
          style={
            Object {
              "padding": "0.5rem",
            }
          }
        >
          <figure
            className="image is-96x96 is-hidden-touch"
          >
            <a
              aria-label="All Yours Buffet"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAABdElEQVQ4y+1Uu0oDURDdL7Gxl/gP+gsWFjY2gmCpjY0glnYWokQ0JCqiJKJoY6GNBA0GFUMem6iIhnUT3OhmH7n3HicbEzZvDZY5cBju3DuHMzPsSvhnSH3BGoQQtVghr8v15LBVYSex3zm0v8A+ZGIKolT8u8OaEKcWKXIjByt9AjMVAtPfwSnJOWvbvtS1VfsTUMKAFqNDqWv7Uv0CyBWnIsGQUxVk315hGTqeM0nEY3eIPdwjehNBJp1y3jmkGreG1NTqz3lrJ4iFpWVEbhM4uwhjbn4RE5PTmJqZxao30NapVHVWBtcysBJ7KMkhKNcbeDxfobgJUz5CNuLHy+U6lCsv8lEfbJqpFd8FLzzBrVERpCGXYSX2oXkHofmGoAeGYWx7UDwYgX48But0HGZwFLqf7vweFHweaGsDsOVDuDXqZ8hMZ6vCyIMXc2BE7lB1Nswo8mqu/MZQnZqOM+wFLWbo/sx4A0UDm+9bOuz/vnrGN3QCEqFdInL+AAAAAElFTkSuQmCC"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/AllYours-3687195526522c748ae7c4e228c3cefb-4c0d3.webp 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-98cef.webp 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/AllYours-3687195526522c748ae7c4e228c3cefb-2a2e7.png 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ce27d.png 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-4c0d3.webp 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-98cef.webp 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-2a2e7.png 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ce27d.png 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
          <figure
            className="image is-48x48 is-hidden-desktop"
          >
            <a
              aria-label="All Yours Buffet"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAABdElEQVQ4y+1Uu0oDURDdL7Gxl/gP+gsWFjY2gmCpjY0glnYWokQ0JCqiJKJoY6GNBA0GFUMem6iIhnUT3OhmH7n3HicbEzZvDZY5cBju3DuHMzPsSvhnSH3BGoQQtVghr8v15LBVYSex3zm0v8A+ZGIKolT8u8OaEKcWKXIjByt9AjMVAtPfwSnJOWvbvtS1VfsTUMKAFqNDqWv7Uv0CyBWnIsGQUxVk315hGTqeM0nEY3eIPdwjehNBJp1y3jmkGreG1NTqz3lrJ4iFpWVEbhM4uwhjbn4RE5PTmJqZxao30NapVHVWBtcysBJ7KMkhKNcbeDxfobgJUz5CNuLHy+U6lCsv8lEfbJqpFd8FLzzBrVERpCGXYSX2oXkHofmGoAeGYWx7UDwYgX48But0HGZwFLqf7vweFHweaGsDsOVDuDXqZ8hMZ6vCyIMXc2BE7lB1Nswo8mqu/MZQnZqOM+wFLWbo/sx4A0UDm+9bOuz/vnrGN3QCEqFdInL+AAAAAElFTkSuQmCC"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/AllYours-3687195526522c748ae7c4e228c3cefb-4c0d3.webp 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-98cef.webp 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/AllYours-3687195526522c748ae7c4e228c3cefb-2a2e7.png 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ce27d.png 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-4c0d3.webp 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-98cef.webp 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-2a2e7.png 200w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-ce27d.png 400w,
/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/AllYours-3687195526522c748ae7c4e228c3cefb-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
        </div>
        <div
          style={
            Object {
              "padding": "0.5rem",
            }
          }
        >
          <figure
            className="image is-96x96 is-hidden-touch"
          >
            <a
              aria-label="What a Save Esports"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAABeUlEQVQ4y9VVPavCQBCMHyCkUbGxEUUlRAlCIKUgqJWFBGKlAYOKprYQrFIKAbGw8tfOY+4R9cAXTrF5xZDjsjs3uzfZaJqm4cv4XVSrVbTbbTSbzbfAHOZKhNy4Xq9Yr9cIwxC73U4Zm80Gl8sF9Xr9QchTSPZpmYvFAt1u90FI6VTGdT6fRzabVQJjmRMEAUzTlAkpn2sGqirLZDLiuVwu0el0/ish+8LAJPhjwuRS3sGfhK1WS9gml8uh3+8LTzmOc0+sVCrvE9KgXN9uN+z3e5zPZxyPR5xOJ+Ez13Uxm82kPqeWvN1uxcZwOEStVoNlWRgMBphMJhiNRuj1erBtWyJKVUhzlstlRFGEw+GA1WoFz/MwnU6fg6XLUlJIdePxWCgiDMMQB72ylJJtisWieBYKBSmZZImdEvASX356z8OhVCoJMl3Xle3j+748HDi+OILm87l4mYClpIHKGBfHMRqNhjxg6T2ewl6ogmUyh2RP/f3uL+AHEFMNtc19XcgAAAAASUVORK5CYII="
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-4c0d3.webp 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-98cef.webp 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-2a2e7.png 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ce27d.png 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-4c0d3.webp 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-98cef.webp 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-2a2e7.png 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ce27d.png 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
          <figure
            className="image is-48x48 is-hidden-desktop"
          >
            <a
              aria-label="What a Save Esports"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAABeUlEQVQ4y9VVPavCQBCMHyCkUbGxEUUlRAlCIKUgqJWFBGKlAYOKprYQrFIKAbGw8tfOY+4R9cAXTrF5xZDjsjs3uzfZaJqm4cv4XVSrVbTbbTSbzbfAHOZKhNy4Xq9Yr9cIwxC73U4Zm80Gl8sF9Xr9QchTSPZpmYvFAt1u90FI6VTGdT6fRzabVQJjmRMEAUzTlAkpn2sGqirLZDLiuVwu0el0/ish+8LAJPhjwuRS3sGfhK1WS9gml8uh3+8LTzmOc0+sVCrvE9KgXN9uN+z3e5zPZxyPR5xOJ+Ez13Uxm82kPqeWvN1uxcZwOEStVoNlWRgMBphMJhiNRuj1erBtWyJKVUhzlstlRFGEw+GA1WoFz/MwnU6fg6XLUlJIdePxWCgiDMMQB72ylJJtisWieBYKBSmZZImdEvASX356z8OhVCoJMl3Xle3j+748HDi+OILm87l4mYClpIHKGBfHMRqNhjxg6T2ewl6ogmUyh2RP/f3uL+AHEFMNtc19XcgAAAAASUVORK5CYII="
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-4c0d3.webp 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-98cef.webp 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-2a2e7.png 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ce27d.png 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-4c0d3.webp 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-98cef.webp 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-2a2e7.png 200w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-ce27d.png 400w,
/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/WhatASave-8bdaedcd8c12348e6af09a181e667097-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
        </div>
        <div
          style={
            Object {
              "padding": "0.5rem",
            }
          }
        >
          <figure
            className="image is-96x96 is-hidden-touch"
          >
            <a
              aria-label="Egging Grocers"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAAB5klEQVQ4y6VU207bQBD1V1c88s4DL7QPUNEiQYFSVQW1EFRuUriVpqQCqkoV5RIpgmB8S3DAsb2nM0PXwpdAIkZazXq9e+bMzNk1kDGlVOKL5k+ZUbT48LBScQKo59k9jwLqjWEcIFQBngrYM2ArtHDQ/AqrU4fbadC4hNO5wF3s98eQLabU2E7bVaxej6HiLuC7+wkb1luc+PupPT0x1NGZDYNU3Hnse18I/LUw7buG6n90m9LdsCYo9RVJv+otEbOoN8CH3YtUKHMvNLFlT8vYdT5gk3wQt/PygipmWBSVAfzIkyDsdf10FlmN5hj6kSvN0Ie1NYltL/Ix9A/dtcPmKhYaQ6i1D6kRi/hzsysN4frtUaf/+hX8aq3j2N9LMdTgRlYqDFAyXwrIZP2FzN/VB7BsvpIucx3fXwxi7Xo8d5NSDHVNardHormr4Azb9izp7ge+OR9J6LbUkQNXifnvVrl7DfVCQLpbt94knf3pLaNsT1GAeRE3p71lz0gALgf7LKiRXWC9rZij2HHmcHl3LDeFmXKAsjWFz41hHDXXpHGlq5GcLo2ijulNsp54lfqXfd4KU869gXJdHKjaOWCSbCw7BZ492/WmJIO/Y+piFN37OE7/V3lg47HXt9t97es9fK79A/OzCt9RgmwIAAAAAElFTkSuQmCC"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-4c0d3.webp 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-98cef.webp 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-2a2e7.png 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ce27d.png 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-4c0d3.webp 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-98cef.webp 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-2a2e7.png 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ce27d.png 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
          <figure
            className="image is-48x48 is-hidden-desktop"
          >
            <a
              aria-label="Egging Grocers"
              href="https://google.com"
              onClick={[Function]}
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "paddingBottom": "100%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAA7EAAAOxAGVKw4bAAAB5klEQVQ4y6VU207bQBD1V1c88s4DL7QPUNEiQYFSVQW1EFRuUriVpqQCqkoV5RIpgmB8S3DAsb2nM0PXwpdAIkZazXq9e+bMzNk1kDGlVOKL5k+ZUbT48LBScQKo59k9jwLqjWEcIFQBngrYM2ArtHDQ/AqrU4fbadC4hNO5wF3s98eQLabU2E7bVaxej6HiLuC7+wkb1luc+PupPT0x1NGZDYNU3Hnse18I/LUw7buG6n90m9LdsCYo9RVJv+otEbOoN8CH3YtUKHMvNLFlT8vYdT5gk3wQt/PygipmWBSVAfzIkyDsdf10FlmN5hj6kSvN0Ie1NYltL/Ix9A/dtcPmKhYaQ6i1D6kRi/hzsysN4frtUaf/+hX8aq3j2N9LMdTgRlYqDFAyXwrIZP2FzN/VB7BsvpIucx3fXwxi7Xo8d5NSDHVNardHormr4Azb9izp7ge+OR9J6LbUkQNXifnvVrl7DfVCQLpbt94knf3pLaNsT1GAeRE3p71lz0gALgf7LKiRXWC9rZij2HHmcHl3LDeFmXKAsjWFz41hHDXXpHGlq5GcLo2ijulNsp54lfqXfd4KU869gXJdHKjaOWCSbCw7BZ492/WmJIO/Y+piFN37OE7/V3lg47HXt9t97es9fK79A/OzCt9RgmwIAAAAAElFTkSuQmCC"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "cover",
                      "objectPosition": "center",
                      "opacity": 0,
                      "position": "absolute",
                      "top": 0,
                      "transition": "opacity 0.5s",
                      "transitionDelay": "0.5s",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-4c0d3.webp 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-98cef.webp 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ff64f.webp 500w"
                    type="image/webp"
                  />
                  <source
                    sizes="(max-width: 500px) 100vw, 500px"
                    srcSet="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-2a2e7.png 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ce27d.png 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png 500w"
                  />
                  <img
                    alt=""
                    onLoad={[Function]}
                    src="/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "cover",
                        "objectPosition": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 0.5s",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source type='image/webp' srcSet=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-4c0d3.webp 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-98cef.webp 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ff64f.webp 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><source srcSet=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-2a2e7.png 200w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-ce27d.png 400w,
/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png 500w\\" sizes=\\"(max-width: 500px) 100vw, 500px\\" /><img src=\\"/static/EggingGrocers-fb9f3fc2943062dde49cf696d033ad09-6c215.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </figure>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`renders nothing 1`] = `
<div
  className="level is-mobile"
>
  <div
    className="level-item"
  >
    <p
      className="title is-4"
    >
      Sponsored By
    </p>
  </div>
  <div
    className="level-item"
  />
</div>
`;
